ui/src/app/components/modals.tsx

```diff
@@ -46,7 +47,7 @@ export const Modal = ({width = 450, loading = false, ...props}) => {
   return <ReactModal
     parentSelector={() => document.getElementById('popup-root')}
     isOpen
-    style={{overlay: styles.overlay, content: {...styles.modal, width}}}
+    style={{overlay: styles.overlay, content: props.contentStyle || {...styles.modal, width}}}
```

Comment:
I want it to actually override the entire style instead of adding it onto the defaults. I'll rename to "contentStyleOverride" to make it more clear.

Reply:
It looks like the usage in Animated Modal is also including the defaults. Is there another usage I'm missing?

Reply:
ah, so the `{...styles.modal, width}` bit is still being applied but on the animated.div element instead of the Modal component. Because of that, the style.content needs to be overwritten to apply just the auto margin instead of applying the same `{...styles.modal, width}` again. This is due to the fact that I need to apply the styleSpring (from the react-spring animation library) onto the animated.div element and I can't do that directly onto the ReactModal component.

Reply:
oh, gotcha! Thanks for the clarification